namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/pingdom.tf

```diff
@@ -0,0 +1,19 @@
+provider "pingdom" {
+}
+
+resource "pingdom_check" "prepare-a-case-production-check" {
+  type                     = "http"
+  name                     = "${var.application}"
+  host                     = "/https/${var.prepare-case-domain}"
```

Comment:
I haven't seen this syntax used for `host` before. 

If you check the other `pingdom.tf` files in the namespaces folder, they seem to just put the domain, without specifying any protocol, e.g.

```
$ ls -1 */resources/pingdom.tf | xargs grep host
...
cccd-production/resources/pingdom.tf:  host                     = "claim-crown-court-defence.service.gov.uk"
cccd-staging/resources/pingdom.tf:  host                     = "staging.claim-crown-court-defence.service.justice.gov.uk"
check-my-diary-prod/resources/pingdom.tf:  host                     = "checkmydiary.service.justice.gov.uk"
...
```
That agrees with the documentation for the pingdom provider, here:
https://github.com/russellcardullo/terraform-provider-pingdom#pingdom-check